docs/source/models/gbd2017/causes/causes.rst

```diff
@@ -23,11 +23,11 @@ causes of death or disease are mutually exclusive and collectively exhaustive.
 
     - ... down to the most detailed level
 
-Measures for causes include:  
-Deaths, incidence, prevalence, YLDs, YLLs, and DALYs
+*Measures* are a quantity of disease burden. Measures for causes include:  
+deaths, incidence, prevalence, remission, excess mortality, YLDs, YLLs, and DALYs
 
-And metrics for causes include:  
-Counts, rates, and fractions
+*Metrics* are units that describe disease burden. Metrics for causes include:  
+counts, rates, and fractions
```

Comment:
You later have a bullet with percentages rather than fractions---either seems fine, but I think it's helpful to choose one